ui/src/app/models/permission-level.ts

```diff
@@ -0,0 +1,7 @@
+// Levels of authorization / permission for CDRs.
+
+export enum PermissionLevel {
+  Public = 1,
```

Comment:
OK. This particular code will probably go away as soon as we start getting info from real backends.